content/articles/integrating-facebook-ads-into-your-android-application-in-android-studio/index.md

```diff
@@ -0,0 +1,172 @@
+android:orientation="vertical"
+App:layout_constraintBottom_toBottomOf="parent"/>
+ 
+</RelativeLayout>
+
+```
+
+On your java activity, implement the Banner Ad by instantiating an Adview object. to do that, place this code in the `onCreate()` function.
+
+```kotlin
+adView = new AdView(this, "IMG_16_9_App_INSTALL#2029572424039676_2029575434039375", AdSize.BANNER_HEIGHT_60);
+ 
+// Find the AdContainer
+LinearLayout adContainer = (LinearLayout) findViewById(R.id.banner_container);
+ 
+adContainer.addView(adView);
+ 
+// Load an ad
+adView.loadAd();
+
+```
+In your activity's `onDestroy()` function, paste the following code. This is done to free up AdView's resources.
+
+```kotlin
+@Override
+protected void onDestroy() {
+if (adView != null) {
+adView.destroy();
+}
+super.onDestroy();
+}
+
+```
+
+With all that done, run your code and observe the output.
+For successful integration, the following output is obtained.
+
+![](engineering-education/integrating-facebook-ads-with-your-android-application-in-android-studio
+/output.png)
+
+You can alter the code to and other types of Ads like interstitial Ads and Native Ads in the same manner.
+
+### Why integrate Facebook ads?
+Consumers prefer in-App advertising over display advertising on mobile websites. Smartphone consumers find ads on the mobile web to be compacted and unpleasant. In-App advertisements are resized to fit the screen and seem better, increasing overall engagement.
+
+When establishing in-App advertisements, **accessibility** is another important issue to consider. In-App mobile advertising is easier to reach because most consumers have their phones with them all the time, wherever they go. You have access to a captive audience or users who are already using the App and are interested in what you have to say. It's easier to convert them because they're already interested.
+
+### Demerits of integrating into some Applications
+In-App advertisement as seen earlier has many advantages. Besides, it is not always Appealing to some users when ads keep popping in all the time.
+
+Here are some demerits in an in-App advertisement;
+
+- People/users might hate the App due to **disturbances from ads pop-up**. This may discourage users from using known Applications with in-App advertisements.
+
+- For some people, it may be **difficult to navigate**. Some interested customers may not get to see the advertisement due to the incompatibility of the ads with some phones.
+
+- **Cost incurred on the user** to get the advertisement. To get the advertisement, the user's internet bundles get consumed from an unintended advertisement viewing task.
+
+### Conclusion
+At this point, you should be able to integrate your application with Facebook Ads for added monetary advantage. However, this might not be an option for many developers depending on the kind of application they are developing. 
+The verdict on whether to integrate your application with these Ads is always the owners' preference. 

```

Comment:
The owner in what context are you taking about here. Please rephrase this sentence.